doc/tutorial/ranges/views/minimiser/index.md

```diff
@@ -0,0 +1,161 @@
+proven to work especially well on repeat regions.
+
+# Usage in SeqAn3
+
+The minimisers are implemented in `seqan3::views::minimiser_hash`. The function requires three arguments: The sequence,
+the shape and the window size. The above-mentioned k-mer size is implicitly given by the size of the shape.
+That is all the information you need to obtain minimisers with SeqAn3, so let's just dive right into the first
+assignment!
+
+\assignment{Assignment 1: Fun with minimisers I}
+Task: Obtain the minimisers for "CCACGTCGACGGTT" with an ungapped shape of size 4 and a window size of 8.
+\endassignment
+\solution
+\include doc/tutorial/ranges/views/minimiser/minimiser_solution1.cpp
+\endsolution
+
+If you have completed the assignment, you probably wonder what these large numbers mean. As explained above, the
+lexicographical ordering is less optimal, therefore, SeqAn3 uses a seed to randomize the order. To do so, SeqAn3 simply
+XORs the hash values with a random seed (Default: 0x8F3F73B5CF1C9ADE). How would you get back the actual hash values
+then?
+Well, you just use XOR again!
+
+\include doc/tutorial/ranges/views/minimiser/seed_example.cpp
+
+From these hash values, you can obtain the sequence they are representing by transforming the numbers to base 4. (For
+example, 182 is "2312" in base four and therefore represents "GTCG".)
+
+Now take a closer look at the resulting minimiser sequences. Are they what you would
+expect? Probably not, these are not the values presenting the minimisers shown in the example above. Why not?
+
+\hint
+The example above is based on a lexicographical ordering, our first assignment is not.
+\endhint
+
+So, what would we need to change to achieve the results from our example? We need to use a different seed. SeqAn3
+makes this simple by allowing us to add another input parameter of type `seqan3::seed`.
+
+\assignment{Assignment 2: Fun with minimisers II}
+Task: Reproduce the results from the example above by obtaining the minimisers for "CCACGTCGACGGTT".
+
+To do so, you have to know which seed to use.
+
+\hint
+Because the randomization is based on an XOR, you have to use a value X which XOR-ed with a hash value results in the
+same hash value.
+\endhint
+
+\hint
+10001 XOR 0 = 10001
+\endhint
+
+\endassignment
+\solution
+\include doc/tutorial/ranges/views/minimiser/minimiser_solution2.cpp
+\endsolution
+
+### Ignoring the backward strand
+
+So far, we have always considered the forward and the backward strand of a sequence, but there might be cases where
+the backward strand should not be considered. If this is the desired behaviour then the `seqan3::views::minimiser` needs
+to be used. Unlike the `seqan3::views::minimiser_hash`, `seqan3::views::minimiser` does not hash the values for you, so
+you have to do this yourself. But fear not, `seqan3::views::kmer_hash` makes this really easy for you!
+
+```cpp
+auto minimisers = text | seqan3::views::kmer_hash(seqan3::ungapped{4}) | seqan3::views::minimiser(5);
+```
```

Comment:
Can you make these into a snippet? You can make one file and several `![snippets]` so its not to much work